compiler+runtime/src/jank/clojure/test.jank

```diff
@@ -0,0 +1,843 @@
+;   Copyright (c) Rich Hickey. All rights reserved.
+;   The use and distribution terms for this software are covered by the
+;   Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+;   which can be found in the file epl-v10.html at the root of this distribution.
+;   By using this software in any fashion, you are agreeing to be bound by
+;   the terms of this license.
+;   You must not remove this notice, or any other, from this software.
+
+;;; test.clj: test framework for Clojure
+
+;; by Stuart Sierra
+;; March 28, 2009
+
+;; Thanks to Chas Emerick, Allen Rohner, and Stuart Halloway for
+;; contributions and suggestions.
+
+;; jank notes:
+;; - use atom instead of ref in *report-counters*
+;; - workaround https://github.com/jank-lang/jank/issues/199
+;;   - wrote -workaround-var->ns to translate var to qualified symbol
```

Comment:
This isn't needed anymore, right?

Reply:
Correct, now we just need `-workaround-var->ns` to workaround #197 and its own implementation does require using a workaround :)

```clojure
 (defn -workaround-var->ns [v]
-  (-> v str (subs 2) symbol namespace symbol the-ns))
+  (-> v symbol namespace symbol the-ns))
```